src/ImmersedBoundaries/immersed_grid_metrics.jl

```diff
@@ -31,3 +33,8 @@ end
 @inline Δzᵃᵃᶜ(i, j, k, ibg::IBG) = Δzᵃᵃᶜ(i, j, k, ibg.underlying_grid)
 @inline Δzᵃᵃᶠ(i, j, k, ibg::IBG) = Δzᵃᵃᶠ(i, j, k, ibg.underlying_grid)
 
+
+return_metrics(grid::ImmersedBoundaryGrid) = return_metrics(grid.underlying_grid)
+min_Δx(grid::ImmersedBoundaryGrid) = min_Δx(grid.underlying_grid)
```

Comment:
Can you make sure here that the minimum is within the fluid region?

Reply:
I thought about that, but do we want to generalize `min_Δz()` to work like that in all cases? I think it would incur some performance penalties since we no longer would be able to just check `grid.Δzᵃᵃᶜ` like we do atm: https://github.com/CliMA/Oceananigans.jl/blob/7c5c5fa61557dcc3d0f8de4b58175ccabf9f4203/src/Grids/rectilinear_grid.jl#L439-L446

We would instead have to check all wet cells (I think?) given that we can't anticipate the shape of the IBM. Maybe we want to create a `min_Δz_wet` instead for that purpose?

Reply:
Agreed! maybe we can leave that to another PR